qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -159,7 +175,14 @@ class CreateCustomerView extends VerticalLayout {
                     break
                 case "affiliation":
```

Comment:
Just from the naming of cases, I don't understand the difference between "affiliation" and the "affiliationValid" case

Reply:
The one is a value of the affiliation field and the other is a boolean describing if this value is valid or not :)